docker/ingestor/core.toml

```diff
@@ -83,8 +83,13 @@ source = '''
     # TODO: support metrics
   }
 
+  # attach a fake token for local mode
+  if is_nullish(.hdx_token) && "${IS_LOCAL_APP_MODE:-false}" == "true" {
+    .hdx_token = "00000000-0000-4000-a000-000000000000"
+  }
+
   # check if token is in uuid format
-  if "${ENABLE_TOKEN_MATCHING_CHECK:-true}" == "true" && !match(to_string(.hdx_token) ?? "", r'^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$') {
+  if !match(to_string(.hdx_token) ?? "", r'^[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$') {
```

Comment:
imo we should disable all checks in local mode, including the UUID one

Reply:
i was debating about this. but I guess its a better experience since people might pass non UUID to auth header 🤔